patches/@types+relay-compiler+8.0.0.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/node_modules/@types/relay-compiler/lib/core/IRPrinter.d.ts b/node_modules/@types/relay-compiler/lib/core/IRPrinter.d.ts
```

Comment:
These changes should probably happen directly in https://github.com/DefinitelyTyped/DefinitelyTyped/tree/9fa257cdddf154838c90557319ade83cdc2a36bf/types/relay-compiler but for the sake of this PR, I chose to only change the types directly here for now.